seedorf/games/models.py

```diff
@@ -181,7 +181,13 @@ class Game(BasePropertiesModel):
     is_featured = models.BooleanField(
         blank=False, default=False, help_text=_("If this game is featured."), null=False, verbose_name=_("Is featured?")
     )
-
+    share_link = models.CharField(
```

Comment:
@Tommos0 : can you make this a URLField as its more indicative of the field content ?